Gems/Atom/RPI/Code/Source/RPI.Public/RPISystem.cpp

```diff
@@ -276,15 +277,10 @@ namespace AZ
             }
         }
 
-        float RPISystem::GetCurrentTime()
+        float RPISystem::GetCurrentTime() const
         {
-            ScriptTimePoint timeAtCurrentTick;
-            AZ::TickRequestBus::BroadcastResult(timeAtCurrentTick, &AZ::TickRequestBus::Events::GetTimeAtCurrentTick);
-
-            // We subtract the start time to maximize precision of the time value, since we will be converting it to a float.
-            double currentTime = timeAtCurrentTick.GetSeconds() - m_startTime.GetSeconds();
-
-            return aznumeric_cast<float>(currentTime);
+            const AZ::TimeUs currentSimulationTimeUs = AZ::GetRealElapsedTimeUs();
```

Comment:
I think we are expecting simulation time here (0 when app starts) but not real time.

Reply:
Real elapsed time is the time since app start and un-modified by time effects (such as t_scale).

Reply:
Checkout ```Code\Framework\AzCore\AzCore\Time\ITime.h``` for the documentation

Reply:
Yes. I am aware of that. I think it's fine we use real time for this case. We will add an another time for the simulation time (time effected by scale fact) in the future.

Reply:
do you want this to be affected by t_scale? we already have that functionality and can easily make the change.
I choose real time as that was what the previous logic was using.

Reply:
I think the real time one is fine. Thank you!